src/components/DocumentRenderer/WebViewFrame.tsx

```diff
@@ -10,22 +10,22 @@ export interface WebViewFrameRef {
   };
 }
 
-export interface WebViewFrameProps {
+export interface WebViewFrame {
   document: Document;
   onTemplateMessageHandler: (event: WebViewMessageEvent) => void;
   setGoToTemplate: Function;
 }
 
-const WebViewFrame: FunctionComponent<WebViewFrameProps> = ({
+export const WebViewFrame: FunctionComponent<WebViewFrame> = ({
   document,
   onTemplateMessageHandler,
   setGoToTemplate
-}: WebViewFrameProps) => {
-  const webViewRef = useRef();
+}: WebViewFrame) => {
+  const webViewRef = useRef<WebView>() as RefObject<WebView>;
```

Comment:
We can avoid casting:
```suggestion
  const webViewRef: RefObject<WebView> = useRef<WebView>(null);
```

Reply:
ah I've tried this but without the null so it was funky, with the null it fixed it. Thanks